app.py

```diff
@@ -562,11 +562,15 @@ def parse():
 
             if transpile_result.has_turtle:
                 response['has_turtle'] = True
+
+            if transpile_result.has_clear:
+                response['has_clear'] = True
         except Exception:
             pass
 
         with querylog.log_time('detect_sleep'):
```

Comment:
@rix0rrr I don't think I understand what is happening here, and why, do you?

Firstly, why the exception? an "in" can't really fail, can it?

Secondly what does the logging do here exactly? Why do we need to log the time that this sleep checking occurs (if I understand correctly)

Reply:
I think the `except` is there for safety. This used to be a function call.

The `log_time` is there to measure how long it takes to do this, in case we need to investigate slowness. With the new code shape, probably not necessary anymore either.